lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```
@@ -367,8 +367,15 @@ project.modal.button.close=Close
 project.list.no.hidden.column.content=No extra content for this project in the following hidden table columns
 
 
-landingPageContent.top=Content at the top of the landing page
-landingPageContent.bottom=Content at the bottom of the landing page
+landingPageContent.authed=Content for authentified users
+landingPageContent.unauthed=Content for unauthentified users
+landingPageContent.unauthed.position=Add content for unauthentified user to authentified users ?
```

```suggestion
landingPageContent.unauthed.position=Add content for unauthentified user to authentified users
```

Usually HIG guidelines don't include question marks. I tried to look some rules, look in your OS settings or in QGIS
https://www.drupal.org/docs/develop/user-interface-standards/radio-buttons